### PR TITLE
Homework6: Nikolai Potovin

### PR DESCRIPTION
**Pull request by adnike**
Bar simulation

### CODE REVIEW

**Comment by adnike** [comment]
Nikolai Potovin

File: src/Homework6.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        print('Бар', self.name)
+        print('Водка', self.vodka_price)
+        print('Пиво', self.beer_price)
+        print('Вино', self.wine_price)
+
+
+class ManInBar(object):
+    def __init__(self, name):
+        self.name = name
+        self.money = 100
```
Comment (belovmd) [inline src/Homework6.py]: why all people have same money amount? it can be set during creation

File: src/Homework6.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        self.ethanol = 0
+        self.danger_level = 'OK'
```
Comment (belovmd) [inline src/Homework6.py]: all this constants should be put somewhere as constants

File: src/Homework6.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        self.card_limit = 1
+
+    def drink(self, type_of_drink, place):
+        if self.danger_level == 'DROP DEAD':
```
Comment (belovmd) [inline src/Homework6.py]: please move to constant

File: src/Homework6.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+            print('No more tonight, wait until tomorrow')
+        else:
+            if type_of_drink == 'vodka':
```
Comment (belovmd) [inline src/Homework6.py]: too high cyclomatic complexity it should be reworked to look more simple

File: src/Homework6.py [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        if self.ethanol >= 0.15:
+            self.danger_level = 'DROP DEAD'
+        elif self.ethanol >= 0.07:
+            self.danger_level = 'DRUNK'
+
+    def info(self):
+        print('И вот перед нами', self.name)
+        print('У него в кармане', self.money)
+        print('Уровень этанола в крови', round(self.ethanol, 3))
+        print('Состояние -', self.danger_level)
```
Comment (belovmd) [inline src/Homework6.py]: as everywhere you use russian it seems it should be mapper from english danger_level to russian one

File: src/Homework6.py [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+    def borrow(self, friend, ammount):
+        if self.danger_level == 'DROP DEAD':
+            print('Sorry, you cant take an action, wait until tomorrow')
+        else:
+            if isinstance(friend, ManInBar):
+                if friend.money - ammount > 0:
```
Comment (belovmd) [inline src/Homework6.py]: should friend borrow all money except one cent?

File: src/Homework6.py [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+                    self.money += ammount
+                    friend.money -= ammount
+                else:
+                    print(friend.name, 'said: "Maaan, its too much')
+
+    def cash(self):
+        if self.card_limit == 1:
```
Comment (belovmd) [inline src/Homework6.py]: what does this 1 number mean? why it mapped to 50?

**Review by belovmd — COMMENTED** [review]
Please add docstrings to class and methods
please address these issues

**Review by belovmd — COMMENTED** [review]
8,5 (maximum score for this task)

**Pull request closed without merging**